tools/macos-installer/productbuild/Resources/en.lproj/conclusion.html.tmpl

```diff
@@ -0,0 +1,18 @@
+<html>
+ <head>
+    <style>
+      body {
+        font-family: Helvetica;
+        font-size: 1.1em;
+      }
+    </style>
+ </head>
+ <body>
+  <p>
+    Node.js was installed at <code>/usr/local/bin/node</code>
```

Comment:
The message looks like this after installation:

<img width="618" alt="screen shot 2017-09-07 at 1 43 13 pm" src="https://user-images.githubusercontent.com/4299420/30147740-8f4fe154-93d2-11e7-9658-e2e7909d254f.png">

Can you wrap each line with `<p>` so they would break better? Also it would be nice to include the version numbers in the message.

Reply:
Like `Node.js {nodeversion} was installed ...`?

Reply:
Maybe something like


```html
<div>
  <p>This package has installed:</p>
  <ul>
    <li>Node.js {nodeversion} to <code>/usr/local/bin/node</code></li>
    <li>npm {npmversion} to <code>/usr/local/bin/npm</code></li>
  </ul>
  <p>Make sure that <code>/usr/local/bin</code> is in your <code>$PATH</code>.</p>
</div>
```

Reply:
TY, let me grab some screenshots of the result

Reply:
Here we go. 
<img width="732" alt="screen shot 2017-09-07 at 08 29 39" src="https://user-images.githubusercontent.com/1814479/30149135-cb30334e-93a6-11e7-9495-7ae5b7c24163.png">

I think we should wrap the unordered list in a paragraph as well to give the last line some space.

Reply:
I would suggest to use css for that because [technically paragraphs can not contain lists](https://www.w3.org/TR/html5/grouping-content.html#the-p-element)...

Reply:
!